src/popups/PopupOptions.as

```diff
@@ -1564,9 +1587,50 @@ package popups
             }
 
             //- Custom Notekin
-            else if (e.target == optionNoteskinsCustom)
+            else if (e.target == optionOpenCustomNoteskinEditor)
+            {
+                navigateToURL(new URLRequest("http://www.flashflashrevolution.com/~velocity/ffrjs/noteskin/"), "_blank");
+                return;
+            }
+
+            //- Custom Notekin
+            else if (e.target == optionImportCustomNoteskin)
+            {
+                var prompt:MultiplayerPrompt = new MultiplayerPrompt(box.parent, _lang.stringSimple("popup_noteskin_import_json")); // "Import JSON"
+                prompt.move(Main.GAME_WIDTH / 2 - prompt.width / 2, Main.GAME_HEIGHT / 2 - prompt.height / 2);
+                prompt.addEventListener(MultiplayerPrompt.EVENT_SEND, function(subevent:Object):void
+                {
+                    try
+                    {
+                        var json:Object = JSON.parse(subevent.params.value);
+                        if (json["rects"] != null && json["data"] != null)
+                        {
+                            // Update Structs
```

Comment:
Which structs? either expand the comment to be more concise or remove it in lieu of just reading the code. ;)